FlexMeasures/flexmeasures
# fix: Inject default colors into global context

## Description

Majorly I injected the color variables into the global context.

Instead of injecting default colors only into the security context (which addresses issues in security templates such as login and password reset), this update injects them into the global context. This ensures that default colors are applied consistently across all templates, including those not rendered by the custom view, such as task templates.

## Look & Feel

![Screenshot from 2024-09-12 11-11-15](https://github.com/user-attachments/assets/37dbf34b-11fa-4049-bbb9-36f53ae8d1d2)
![Screenshot from 2024-09-12 11-11-31](https://github.com/user-attachments/assets/89cb6c42-ff87-4c24-bcec-783f74050c0a)
![Screenshot from 2024-09-12 11-11-40](https://github.com/user-attachments/assets/601610a4-f8da-4370-a514-67c0e5eeab1c)

## How to test

1. Log in again and visit the task page; you shouldn’t see a colorless page.
2. Log out of your account.
3. Visit the login or reset password page.

## Further Improvements

I  would like to investigate why the task page isn't loading from the custom view for template rendering. This issue is preventing it from receiving the computed color values.

## Related Items

Fixes the issue: #1174 

---

- [✅] I agree to contribute to the project under Apache 2 License. 
- [✅] To the best of my knowledge, the proposed patch is not based on code under GPL or other license that is incompatible with FlexMeasures
